src/_common.scss

```diff
@@ -1,9 +1,9 @@
 // COMMON DEPENDENCIES:
-// - This partial loads all functions,mixins,configs and variables.
+// - This partial loads all functions,internal-mixins,configs and variables.
 // - It should be used inside an external project component scope in order to access vibe dependencies.
 
 // Dependencies
 @import "functions";
-@import "mixins";
+@import "internal-mixins";
```

Comment:
the mixins we had so far did not meant to be exported and their purpose was internal usage